Pod/Classes/URBNAlertView.m

```diff
@@ -111,12 +119,14 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
         NSDictionary *metrics = @{@"sectionMargin" : self.alertStyler.sectionVerticalMargin,
                                   @"btnH" : self.alertStyler.buttonHeight,
                                   @"lblHMargin" : self.alertStyler.labelHorizontalMargin,
-                                  @"titleVMargin" : titleMargin,
+                                  @"topVMargin" : @([titleMargin doubleValue] + [self.alertStyler.topMargin doubleValue]),
```

Comment:
Fair points. Backwards compatibility is key so thank you for taking that into account. We will have another quick chat tomorrow and get this in as soon as we can. Thanks for the response.